### PR TITLE
Fix npm test

### PR DESCRIPTION
**Pull request by flenter**

### CODE REVIEW

**Comment by lindseybateman** [comment]
hooray! lgtm

File: docs-learnblocks/index.js [hunk 1]
```diff
@@ -53,9 +53,9 @@ function createClassName (block) {
   const active = (head === slugify(block));
 
   return 'learnblocks-item ' +
-    'learnblocks-item_' +
-    block.replace(/(\.)/gm, '_') +
-    (active ? ' active' : '');
+  'learnblocks-item_' +
```
Comment (hatchan) [inline docs-learnblocks/index.js]: does jsfmt do this?

File: package.json [hunk 2]
```diff
@@ -19,49 +18,36 @@
...
-    "@docs/start": "file:docs-start",
-    "@docs/toc-menu": "file:docs-toc-menu",
-    "@react/icon": "file:react-icon",
-    "@wercker/wercker-animations": "file:wercker-animations",
-    "@wercker/wercker-colors": "file:wercker-colors",
-    "@wercker/wercker-flex": "file:wercker-flex",
-    "@wercker/wercker-fonts": "file:wercker-fonts",
-    "@wercker/wercker-logo": "file:wercker-logo",
-    "@wercker/wercker-typography": "file:wercker-typography",
+    "aws-sdk": "^2.1.17",
```
Comment (hatchan) [inline package.json]: just wondering, why do we need aws-sdk in docs?

Comment (flenter) [inline package.json]: I think it's for redirects?

Comment (hatchan) [inline package.json]: Aah the legacy urls

File: docs-learnblocks/index.js (hunk 1 above)
Comment (flenter) [inline docs-learnblocks/index.js]: semistandard was complaining here.

Comment (hatchan) [inline docs-learnblocks/index.js]: we're not using that anymore, right?

Comment (flenter) [inline docs-learnblocks/index.js]: It's still there. That's what this pr fixes

Comment (flenter) [inline docs-learnblocks/index.js]: The npm test executes semistandard

Comment (hatchan) [inline docs-learnblocks/index.js]: Ehm ok... Then what about replacing it with `util.replace()` looks way better than this concatenation :-/

Comment (flenter) [inline docs-learnblocks/index.js]: I've updated the code and extracted the replace statement.

File: docs-learnblocks/index.js [hunk 3]
```diff
@@ -51,11 +51,11 @@ function render () {
 function createClassName (block) {
   const head = window.location.pathname.split('/')[2];
   const active = (head === slugify(block));
+  const blockName = block.replace(/(\.)/gm, '_');
 
   return 'learnblocks-item ' +
-    'learnblocks-item_' +
-    block.replace(/(\.)/gm, '_') +
-    (active ? ' active' : '');
+  'learnblocks-item_' + blockName +
```
Comment (hatchan) [inline docs-learnblocks/index.js]:
```
util.format('learnblocks-item learnblocks-item_%s%s', blockName, (active ? ' active' : ''));
```

Or move the active related code to blockName

Comment (flenter) [inline docs-learnblocks/index.js]: done. I'm going to merge this in (after it's green).

**Pull request merged**